src/OneExplorer/OneExplorer.ts

```diff
@@ -500,7 +499,17 @@ input_path=${modelName}.${extName}
 
 export function initOneExplorer(context: vscode.ExtensionContext) {
   // TODO Support multi-root workspace
-  let workspaceRoot: vscode.Uri = vscode.Uri.file(obtainWorkspaceRoot());
+  let workspaceRoot: vscode.Uri|undefined = undefined;
+  try {
+    workspaceRoot = vscode.Uri.file(obtainWorkspaceRoot());
+  } catch (e: unknown) {
+    if (e instanceof Error) {
+      Logger.error('OneExplorer', e.message);
+    } else {
+      throw e;
+    }
+  }
+
   // NOTE: Fix `obtainWorksapceRoot` if non-null assertion is false
   const oneTreeDataProvider = new OneTreeDataProvider(workspaceRoot!);
```

Comment:
I wonder what happen here when `(e instanceof Error)`. I guess `workspace` is `undefined` and is it OK to create `TreeDataProvider(undefined)` ?

Reply:
I expected it will be okay since `workspaceRoot!` will block passing `undefined` to the constructor, but it wasn't. I'll fix and add a patch ASAP

Reply:
@dayo, Could you check if `new OneTreeDataProvider(undefined)` is OK or not. Thanks.

Reply:
@hyunsik-yoon 

What happens is : It'll show an information message "Cannot ifnd workspace root" and shows nothing. 

```ts
  getChildren(element?: OneNode): OneNode[]|Thenable<OneNode[]> {
    if (!this.workspaceRoot) {
      vscode.window.showInformationMessage('Cannot find workspace root'); // Shows an information message
      return Promise.resolve([]);
    }
```